python/tvm/relay/ir_pass.py

```diff
@@ -191,6 +191,23 @@ def simplify_inference(expr):
     return _ir_pass.simplify_inference(expr)
 
 
+def simplify_bias_add(expr):
```

Comment:
Let us rename this to canonicalize_ops since bias_add lowering can be viewed as canonicalization step for ops, we might add more op canonicalizations later, (e.g. mean)..